packages/aws-lambda-auto-wrap/esm/utils.js

```diff
@@ -0,0 +1 @@
+../src/utils.js
```

Comment:
_question:_ This looks slightly weird, but probably only in the Github diff. I assume this is not actually a new `.js` file that only contains this single line, right? Is it a symlink? If so, do we need to have it as a symlink here? I guess it is about how we package/copy files in the publish-layer script, is it? Could you explain this a bit?

Reply:
Yes it is a symlink. So the esm/index.js has to require the utility and it cannot do `../src/utils`.
Because when we build the layer zip, the esm handler has no longer access to the location.
That's why we require `./utils` and create a symlink locally.

Do you have a better idea? I wanted to avoid to create a build process such as create a single JS file etc.

Reply:
Thanks for the explanation, that makes sense. I don't have a better idea. We might have more options when we migrate the commonjs wrapper away from being an npm package to just copying it. Maybe we can revisit this then. Fine for now 👍